bin/verify-exercise-example

```diff
@@ -34,23 +34,20 @@ echo "${slug}: testing..."
 # stderr to capture it
 cd "${solution_dir}" || exit
 echo "${solution_dir}"
-test_output=$(sqlite3 '' --init "${slug}_test.sql" -bail  '.exit' 2>&1)
 
-# Check if sqlite ran withour errors:
-if (( $? != 0 )); then
+if ! test_output=$(sqlite3 '' --init "${slug}_test.sql" -bail  '.exit' 2>&1); then
     echo "Test failed for ${slug} - sqlite3 error:"
     echo "${test_output}"
     exit 1
 fi
-[[ -e ./user_output.md ]] && rm ./user_output.md
+rm -f ./user_output.md
 
 # Check if all tests were succesfull
-failed_results=$(jq '.[] | select(.status=="fail") | {name, message}' ./output.json) || exit
-rm ./output.json
```

Comment:
I thought it might be useful for diagnosing problems while developing the example.sql, but I see that the user-focused stdout is all that's needed. 

Also, we're not using `output_dir` for anything.

Will remove.

Reply:
Hmm, the `output.json` file is explicitly created by the slug_test.sql script to be in the current directory. There's no point in having `output_dir` at all.